app/Http/Controllers/ProjectController.php

```diff
@@ -91,4 +96,34 @@ public function delete($id)
         // TODO: Notice of success?
         return redirect('/projectfinder');
     }
+
+    /**
+     * Adds the comment to the given project as long as the user is logged in
+     *
+     * @param $request = user entered text, $project project_id for lookup
+     * @return back to same page
+     */
+    public function addComment(Request $request, Project $project)
```

Comment:
Just a note, we can create custom requests that can contain the validation logic. [See here](https://github.com/uidaho/squireproject/commit/1489809645c661571844facd0f8854cfebe4d2dd#diff-33d60f9cc69d473cb04584baf77d7c4cR7). Probably the _right_ way to do it, but i'm fine with this too.